.changeset/mean-jokes-tan.md

```diff
@@ -0,0 +1,7 @@
+---
+'firebase': patch
+'@firebase/firestore': patch
+'@firebase/firestore-types': patch
+---
+
+feat: Added `inherit` setting to merge existing settings with new settings when calling `firestore.settings()`.
```

Comment:
Maybe something with "the settings from a previous call. This allows customizing custom settings on top of the settings that were applied by `@firestore/testing`. Yadadada."

Reply:
updated.